CRR/clients.js

```diff
@@ -27,7 +29,34 @@ function setupClients({
     secretKey,
     endpoint,
 }, log) {
-    const awsConfig = {
+    const awsv3Config = {
+        region: 'us-east-1',
+        credentials: {
+            accessKeyId: accessKey,
+            secretAccessKey: secretKey,
+        },
+        endpoint,
+        forcePathStyle: true,
+        requestHandler: new NodeHttpHandler({
+            httpAgent: new http.Agent({ keepAlive: true }),
+            httpsAgent: new https.Agent({ 
```

Comment:
FYI there is this project that we try to user when we use custom http agents, to auto enforce consistent keepalive & other rules: https://github.com/scality/httpagent. That might be helpful here.